package.json

```
@@ -6,7 +6,7 @@
   "repository": "https://github.com/metabase/metabase",
   "license": "private",
   "engines": {
-    "node": ">=8.11.4",
+    "node": ">=14.2 <=16.99",
```

Would this look a bit nicer?
```suggestion
    "node": ">=14.2 <17",
```

---

Oh yeah, very good idea. I've adjusted it accordingly.